src/ossia/protocols/midi/midi_protocol.hpp

```diff
@@ -33,27 +33,39 @@ struct OSSIA_EXPORT midi_info
   };
 
   midi_info() = default;
-  midi_info(Type t, std::string d, int p)
-      : type{t}, device{std::move(d)}, port{p}
+  midi_info(Type t, std::string d, std::string n, int p, bool v)
+      : type {t}
+      , device {std::move(d)}
+      , name {std::move(n)}
+      , port {p}
+      , isVirtual {v}
   {
   }
 
-  Type type{};
-  std::string device{};
-  int port{};
+  Type type {};
+  std::string device {};
+  std::string name {};
+  int port {};
+  bool isVirtual {};
```

Comment:
hmmm that's a nitpick but libossia uses snake_case

Reply:
fixed the syntax